spec/stripe_spec.rb

```diff
@@ -25,6 +26,8 @@
   before do
     stub_command('[ `rs_config --get decommission_timeout` -eq 600 ]').and_return(false)
     stub_command("/usr/bin/mysql -u root -e 'show databases;'").and_return(true)
+    stub_command("chown --recursive --silent mysql:mysql #{new_mysql_dir}").and_return(true)
+    stub_command("stat -c %U #{new_mysql_dir}/mysql |grep mysql").and_return(false)
```

Comment:
Since this command will be removed, you will have to stub out the Ruby command that's replacing this.